plugins/android/src/main/java/androidProjectConfiguration.kt

```diff
@@ -6,8 +6,10 @@ import org.gradle.api.JavaVersion
 import org.gradle.api.Project
 import org.gradle.api.artifacts.dsl.RepositoryHandler
 import org.gradle.api.tasks.Delete
+import tools.forma.android.config.FormaConfigurationKey
+import tools.forma.android.config.DefaultConfigurationKey
 
-fun Project.androidProjectConfiguration(
+fun Project.androidProjectDefaultConfiguration(
```

Comment:
Why does need to split method `androidProjectConfiguration` on `androidProjectDefaultConfiguration`? Isn't enough to pass  "default" argument and make single logic for all configuration types?

Reply:
I split it to 2 methods adding ability easily use forma with single configuration. Also second method accepts all params as nullable, and values for null fields are "inherited" from default configuration

Reply:
I saw implementation deeper and don't sure, that we should hold 2 methods for this behavior, case there're 2 entry point for android project configuration: 'default' logic with implicit config type underhood and trough type. It doesn't good for clearly API and brought complexity for supporting it in future.

I thought, that configuration for multiple projects would be:
```
androidProjectConfiguration(
    **type = Mobile,**
    minSdk = 21,
    targetSdk = 29,
    compileSdk = 29,
    kotlinVersion = "1.4.20",
    agpVersion = "4.2.0-beta01",
    versionCode = 1,
    versionName = "1.0",
    dataBinding = true
)

androidProjectConfiguration(
    **type = Tv,**
    minSdk = 23,
    versionCode = 10,
    versionName = "2.0"
)
```
For that, we need:
1. Forma.kt: Deprecate single field for configuration
2. Forma.kt: Added multiple configurations (as it done already)
3. androidProjectConfiguration.kt: Remain one method and add behavior, which will be take `Mobile` configuration from map as default, makes copy and pass values for new project type in it